langlib/langlib-test/src/test/java/org/ballerinalang/langlib/test/LangLibSubTypeTest.java

```diff
@@ -206,7 +206,6 @@ public void testNegativeIntSubType() {
         BAssertUtil.validateError(result, err++, "incompatible types: expected 'decimal'," + FOUND_UNSIGNED_32, 237,
                                   17);
 
-        BAssertUtil.validateError(result, err++, EXPECT_BYTE + FOUND_INT, 251, 14);
```

Comment:
`byte r = 1 | 1;` also seems to be allowed now. Didn't we discuss to raise this in the spec?